CHANGELOG.md

```diff
@@ -147,6 +147,7 @@
 * [ENHANCEMENT] server: clarify documentation for `-server.grpc-max-concurrent-streams` CLI flag. #369
 * [ENHANCEMENT] Ring: Add ID attribute to `InstanceDesc` for ring members. #387
 * [ENHANCEMENT] httpgrpc, grpcutil: added constants and functions for adding request details into outgoing grpc metadata. #391
+* [ENHANCEMENT] Use `net.JoinHostPort` instead of `fmt.Sprintf` to concatenate host and port throughout the codebase to ensure ipv6 compatibility for provided hosts. #388
```

Comment:
I'd rephrase this to focus on the outcome, not the mechanism we use to achieve it, and I think this can be classified as a bug fix rather than an enhancement (in which case, it should be moved to the end of the list):

```suggestion
* [BUGFIX] Correctly format `host:port` addresses when using IPv6. #388
```

Reply:
Done